website/docs/general/blog-contributing-guide.md

```diff
@@ -0,0 +1,353 @@
+---
+id: blog
+title: Blog Contributing Guide
+keywords:
+- API gateway
+- APISIX
+- Apache APISIX
+- blog guide
+description: This article is a set of guidelines for contributors who want to write blogs. This guideline will teach you how to contribute if you want to add a new blog, or modify existing blog contents.
+---
+
+## Overview
+
+This guideline will teach you how to contribute if you want to add a new blog, or modify existing blog contents.
+
+If you find an issue on current documents, please feel free to [file an issue](https://github.com/apache/apisix-website/issues/new) and let the community know about it, or you can [sumbit a pull request](https://github.com/apache/apisix-website/pulls) to fix or update. Both actions are welcome and recommended.
+
+The blogs are written in two languages: English and Chinese. We encourage contributors to add blogs in their preferred language. It is completely up to you. We can handle translations and ask you to do a pull request review later.
+
+English blogs are located in the `website/blog` directory, in which they are categorized by year, month and date. For example, `website/blog/2021/11/22/develop-apisix-ingress-with-nocalhost-in-kubernetes.md` means that a blog named `develop-apisix-ingress-with-nocalhost-in-kubernetes.md` was published on November 22nd, 2021, and it is located in the `website/blog/2021/11/22 directory`.
+
+Similarly, Chinese blogs are located in `website/i18n/zh/docusaurus-plugin-content-blog` directory and follow the same rules described above.
+
+## What you can contribute
+
+We encourge contributors and users to write blogs about Apache APISIX and publish them on the website. As long as blogs fit in the catogories listed below:
```

Comment:
`As long as` is a little limitation IMHO, could we receive other kinds of blog contents?